cmd/wrap.go

```diff
@@ -40,13 +41,17 @@ This allows you to do this:
 		bin := args[0]
 		binargs := args[1:]
 
+		kubeconfig, err := getKubeconfig()
```

Comment:
yes, the "" case was intentional to allow default config to be used. I worry if truss wrap command behaves different from the other ones

Reply:
The reasoning behind this change is to prevent the possibility of unintentionally running a command in the wrong env because you _thought_ you set things to edge with `-e cmh-edge`, but that's not right (it's `edge-cmh`) and it might use a previous KUBECONFIG that points to prod or whatever because it falls back to inheriting that from your env.

_However_ I could buy the case where you don't pass the `-e` flag is different and explicitly expresses the intention to inherit from whatever your KUBECONFIG is.

I decided against ☝️  because... why use `truss wrap` at all in that case? But I could be convinced.